ksqldb-execution/src/main/java/io/confluent/ksql/execution/util/ColumnExtractor.java

```diff
@@ -13,7 +13,7 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package io.confluent.ksql.analyzer;
+package io.confluent.ksql.execution.util;
```

Comment:
I don't love the package this was moved to but couldn't think of anywhere better. Suggestions welcome!